includes/Story_Post_Type.php

```diff
@@ -275,6 +275,12 @@ public static function admin_enqueue_scripts( $hook ) {
 
 		self::load_admin_fonts( $post );
 
+		// Media settings.
+		$max_upload_size = wp_max_upload_size();
+		if ( ! $max_upload_size ) {
+			$max_upload_size = 0;
```

Comment:
Should we have a default here instead of `0`? Is the intention not to allow uploads in this case?

Reply:
This code is copied from [here](https://github.com/WordPress/gutenberg/blob/e33a49f691e60685bf2553095fd2edac860a535b/lib/edit-site-page.php#L36-L40). All this code does it make sure that value is int. If 0, that is what is it, it is possible.

Reply:
> If 0, that is what is it, it is possible.

Not sure I understand what this means.

Reply:
`wp_max_upload_size` can return `0` as `ini_get( 'upload_max_filesize' )` or `ini_get( 'post_max_size' )` could be set to `0`. Unlikely but possible. This code makes sure if `wp_max_upload_size()` returns a falsy value, it makes it is an int of `0`.

Reply:
Yep, understood that. What I'm wondering is -- what's the expected behavior when the value is `0`? Would this mean that the file would always be too large for uploading?

Reply:
Setting to `0` basically means disabling uploads. This check would mean every file would fail to upload. Which is the only / correct behavior.

Reply:
Okay, that's what I wanted to confirm, if that was intended! Thanks.